coforma/swift-tech-challenge
# Issue/404

# Updates 404 page

# Summary
Updated default USWDS 40 to match Figma mockup

## Type of change

# How To Test
1. Navigate to a route that does not exist
2. See the new 404 page

# Checklist:
